### PR TITLE
Showcase

### PR DESCRIPTION
**Pull request by johnsja**
Adding a bootstrap carousel based showcase. This configurable option will replace the homepage image by default but the view html is easy to refactor to meet the needs of other/additional use cases.

Images and youtube videos may be added to the showcase via an admin interface. Images are saved via Active Storage.

### CODE REVIEW

File: app/views/admin/showcase_items/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<%= simple_form_for [:admin, @item] do |f| %>
+  <div class="form-group">
+    <label>Name</label>
```
Comment (cdinger) [inline app/views/admin/showcase_items/_form.html.erb:3]: These labels are detached from their form controls, which hurts accessibility. We should either manually add the `for="showcase_item_name"` attribute here (and on the others) or use the built-in labels. You can pass custom label text to these like this if you want:

```erb
<%= f.input :name, label: "Name <small><i>(Administrator reference only; will not appear anywhere.)</i></small>".html_safe %>
```

File: app/controllers/admin/showcase_items_controller.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    @item = ShowcaseItem.find(params[:id])
+
+    add_breadcrumb 'Showcase', :admin_showcase_items_path
+    add_breadcrumb 'Edit Item'
+  end
+
+  def update
+		@item = ShowcaseItem.find(params[:id])
+
+		if @item.update(item_params)
```
Comment (cdinger) [inline app/controllers/admin/showcase_items_controller.rb:36]: Indentation got a little wacky here

**Review by cdinger — CHANGES_REQUESTED** [review]
Thanks for putting this together, @johnsja! I added some comments about form labels and ActiveStorage configuration.

File: config/storage.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+local:
```
[the remark this answers is not in the document]
Comment (johnsja) [reply config/storage.yml]: I've updated the storage services based on your feedback. Local (development) and test environments will now use the "temp" service as assets are unimportant and can be discarded. Staging and production environments will use the "persistent" service.

File: app/controllers/admin/showcase_items_controller.rb (hunk 2 above)
Comment (johnsja) [reply app/controllers/admin/showcase_items_controller.rb]: Fixed. Not sure why that happened.

File: config/storage.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (cdinger) [reply config/storage.yml]: I like it.

File: app/views/admin/showcase_items/_form.html.erb (hunk 1 above)
Comment (johnsja) [reply app/views/admin/showcase_items/_form.html.erb]: I must have not seen this comment the other day but this has now been addressed. I was following the pattern the other admin interfaces set but you're right.

**Comment by cdinger** [comment]
Thanks, @johnsja. I like the use of the `hint` key. Can you also add a label to the upload control?

**Review by cdinger — APPROVED** [review]

**Pull request merged**